### DISCUSSION TITLE
Can an Wikidata instance be used as a predicate in a RDF triple?

### DISCUSSION THREAD

**Discussion by Yueqiao12Zhang** [DDMAL/linkedmusic-datalake#42] [Q&A] [answered]
For example:

In Wikidata, if we search "mode", we only have a [(https://www.wikidata.org/wiki/Q731978)](url), which is an instance but not a property. In my understanding, an instance is similar to a "noun", and is often used as a subject or an object. But if we use an instance as a predicate, it does not really make sense.

**Comment by fujinaga** [comment, marked as answer]
I think that issue has come up before (read previous Discussions). Anyways, that's not your concern. The task at hand is to take already reconciled CSV (via OpenRefine or manually) then imported into Virtuoso in Quad RDF. If you need a place holder, you can certainly use Wikidata Q-instance. Perhaps you can create a new issue with the title: Import reconciled CSV into Virtuoso. if you need to first convert the CSV into Turtle, create a separate issue and document why this is necessary, rather than going directly into Virtuoso.
